src/view/components/toolbar-add-contentlet/toolbar-add-contentlet.html

```diff
@@ -0,0 +1,25 @@
+<dot-dropdown-component
+        icon="add"
+        alignRight="true"
```

Comment:
By the way... I want to do a refactor here, the option in the dropdown-component should be like ```align="left"``` alignRight or alignLeft it's not cool.